dist/index.js

```diff
@@ -7844,53 +7844,54 @@ exports.request = request;
-
-    // Create a release
-    // API Documentation: https://developer.github.com/v3/repos/releases/#create-a-release
-    // Octokit Documentation: https://octokit.github.io/rest.js/#octokit-routes-repos-create-release
-    const createReleaseResponse = await github.repos.createRelease({
-      owner,
-      repo,
-      tag_name: tag,
-      name: releaseName,
-      draft,
-      prerelease
-    });
-
-    // Get the ID, html_url, and upload URL for the created Release from the response
-    const {
-      data: { id: releaseId, html_url: htmlUrl, upload_url: uploadUrl }
-    } = createReleaseResponse;
-
-    // Set the output variables for use by other actions: https://github.com/actions/toolkit/tree/master/packages/core#inputsoutputs
-    core.setOutput('id', releaseId);
-    core.setOutput('html_url', htmlUrl);
-    core.setOutput('upload_url', uploadUrl);
-  } catch (error) {
-    core.setFailed(error.message);
-  }
-}
-
-module.exports = run;
+const core = __webpack_require__(470);
+const { GitHub, context } = __webpack_require__(469);
+
+async function run() {
+  try {
+    // Get authenticated GitHub client (Ocktokit): https://github.com/actions/toolkit/tree/master/packages/github#usage
+    const github = new GitHub(process.env.GITHUB_TOKEN);
+
+    // Get owner and repo from context of payload that triggered the action
+    const { owner, repo } = context.repo;
+
+    // Get the inputs from the workflow file: https://github.com/actions/toolkit/tree/master/packages/core#inputsoutputs
+    const tagName = core.getInput('tag_name', { required: true });
+
+    // This removes the 'refs/tags' portion of the string, i.e. from 'refs/tags/v1.10.15' to 'v1.10.15'
+    const tag = tagName.replace('refs/tags/', '');
+    const releaseName = core.getInput('release_name', { required: true }).replace('refs/tags/', '');
+    const draft = core.getInput('draft', { required: false }) === 'true';
+    const prerelease = core.getInput('prerelease', { required: false }) === 'true';
+
+    // Create a release
+    // API Documentation: https://developer.github.com/v3/repos/releases/#create-a-release
+    // Octokit Documentation: https://octokit.github.io/rest.js/#octokit-routes-repos-create-release
+    const createReleaseResponse = await github.repos.createRelease({
+      owner,
+      repo,
+      tag_name: tag,
+      name: releaseName,
+      draft,
+      prerelease
+    });
+
+    // Get the ID, html_url, and upload URL for the created Release from the response
+    const {
+      data: { id: releaseId, html_url: htmlUrl, upload_url: uploadUrl }
+    } = createReleaseResponse;
+
+    // Set the output variables for use by other actions: https://github.com/actions/toolkit/tree/master/packages/core#inputsoutputs
+    core.setOutput('id', releaseId);
+    core.setOutput('html_url', htmlUrl);
+    core.setOutput('upload_url', uploadUrl);
+    core.setOutput('tag', tag);
+  } catch (error) {
+    core.setFailed(error.message);
+  }
+}
+

```

Comment:
What happened here? Spaces to tabs or vice-versa?

Reply:
`dist/index.js`  Is this file auto-generated by build?   @monkeyWie  Can you help fix it?

Reply:
Should  I revert`dist / index.js` and build it by author self?

Reply:
@IAmHughes  help merge this pr, pls.

Reply:
@IAmHughes   Help to merge this pr, pls.

Reply:
@deancn `dist/index.js` is created and added to the commit by `npm run precommit`, which is run by a pre-commit Git hook added by husky when you `npm install` the project. You should not edit it yourself, but instead fix errors in `src/`.